### ISSUE TITLE
Consistent focus outline styles across core and plugins

### ISSUE THREAD

**Issue by dopenguin** [Dataport/polar#871]
## Description

The focus outline styles currently deviate in a few places. They should be streamlined, so only two focus outline styles are used:
- Elements that are being displayed *directly* over the map (e.g. `PolarIconButton`)
- Elements that are being used in a context that include a background

## Current behavior

Outlines are sometimes styled by KERN, sometimes they use the native outline styles and in the case of the `PolarIconButton` we use completely different approach.

## Expected behavior

Any element that is being displayed directly on the map should use the focus outline styling of the `PolarIconButton`.
Any other element should use the KERN default focus outline.

```css
outline: 4px solid var(--kern-color-action-focus-default-contextual, #454B6B);
outline-offset: 2px;
```

## Out of scope

Only the focus style should be changed, no logic.
